src/dialects/abstract/query-generator.js

```diff
@@ -244,7 +260,12 @@ class QueryGenerator {
       emptyQuery += returningFragment;
     }
 
-    query = `${replacements.attributes.length ? valueQuery : emptyQuery};`;
+    if (this._dialect.supports.returnIntoValues && options.returning) {
+      // Populating the returnAttributes array and performing operations needed for output binds of insertQuery 
+      this.getInsertQueryReturnIntoBinds(returnAttributes, bind.length, returningModelAttributes, returnTypes, options);
```

Comment:
For the Oracle dialect we generate sql statement like these - `INSERT INTO "Campuses" ("id","name","address","description","createdAt","updatedAt") VALUES (DEFAULT,:1,:2,:3,:4,:5) RETURNING "id","name","address","description","createdAt","updatedAt" INTO :6,:7,:8,:9,:10,:11;`

`getInsertQueryReturnIntoBinds` function generates the bind values for outBinds (:6,:7,:8,:9,:10,:11 in the above sql) , generates the bindDefs for the out binds and also populates the options.outBindAttributes field.

Yes, this function is only needed by the Oracle dialect, for now, we have defined an abstract function with a noop so that other dialects can override it if required.